src/ImportDefinitionsBundle/Interpreter/DefaultObject.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Import Definitions.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2017 Divante (http://www.divante.co)
+ * @license    https://github.com/w-vision/ImportDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ */
+
+namespace ImportDefinitionsBundle\Interpreter;
+
+use ImportDefinitionsBundle\Model\DefinitionInterface;
+use ImportDefinitionsBundle\Model\Mapping;
+use Pimcore\Model\DataObject;
+use Pimcore\Model\DataObject\Concrete;
+
+/**
+ * Class DefaultObject
+ *
+ * @package ImportDefinitionsBundle\Interpreter
+ */
+class DefaultObject implements InterpreterInterface
```

Comment:
I would'nt name it DefaultObject, as it expects to let you configure a "default" object. Something like ObjectByPath seems more right.